client/src/components/Form/Elements/FormDrilldown/FormDrilldownOption.vue

```diff
@@ -0,0 +1,74 @@
+<script setup lang="ts">
+import { computed, onMounted, ref, type ComputedRef } from "vue";
+import { BFormCheckbox, BFormRadio } from "bootstrap-vue";
+import { getAllValues, type Option } from "./utilities";
+import FormDrilldownList from "./FormDrilldownList.vue";
+
+export interface Props {
```

Comment:
same here, we probably don't need to `export` this, unless it's being imported elsewhere

Reply:
Thank you!